src/components/app-content-switcher/AppContentSwitcher.tsx

```diff
@@ -14,14 +14,16 @@ interface SwitcherProps {
   switchOptions: SwitchOptions
   typographyVariant: TypographyProps['variant']
   styles?: SxProps
+  checked?: boolean
```

Comment:
`checked` is a value, `defaultChecked` is a initial value
```suggestion
  defaultChecked?: boolean
```

Reply:
agree , fixed